metricbeat/module/windows/perfmon/pdh_windows.go

```diff
@@ -109,7 +116,15 @@ func (q *Query) AddCounter(counterPath string) error {
 		return errors.Wrapf(err, `failed to add counter (path="%v")`, counterPath)
 	}
 
-	q.counters[counterPath] = h
+	q.counters[counterPath] = &Counter{handle: h}
+	switch format {
+	case "float":
+		q.counters[counterPath].format = PdhFmtDouble
+	case "uint64":
```

Comment:
Shouldn't these actually be called `int64` and `int32`? I didn't see any mention that the values are unsigned in the docs (possibly I missed it). And you do cast to `int64` and `int32` later in the code.